storage/api/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteStorageManager.java

```diff
@@ -123,7 +123,7 @@ InputStream fetchLogSegment(RemoteLogSegmentMetadata remoteLogSegmentMetadata,
      * Returns the index for the respective log segment of {@link RemoteLogSegmentMetadata}.
      * <p>
      * Note: The transaction index may not exist because of no transactional records.
```

Comment:
I am fine to leave this comment for better clarity even though [param doc](https://github.com/apache/kafka/pull/14363/files#diff-39904d4df047525df5309abf04c33a73e7ecb74e175d81729bc17ffd7c7067bdR132) clarifies that `RemoteResourceNotFoundException` is thrown when any index is not found in remote storage.

Reply:
Good to me, too.